libraries/rush-lib/src/schemas/experiments.schema.json

```diff
@@ -53,6 +53,10 @@
     "forbidPhantomResolvableNodeModulesFolders": {
       "description": "If true, Rush will not allow node_modules in the repo folder or in parent folders.",
       "type": "boolean"
+    },
+    "enablePnpmSyncForInjectedDependenciesMeta": {
+      "description": "If true, Rush will use pnpm-sync to manage injected dependenciesMeta in PNPM.",
+      "type": "boolean"
     }
```

Comment:
Add your setting to the `rush init` template for this file:

https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/assets/rush-init/common/config/rush/experiments.json